src/pages/organizations/[organizationid].js

```diff
@@ -356,24 +343,20 @@ export default function Organization(props) {
             >
               Species of trees planted
             </Typography>
-            <Box
-              sx={{
-                mt: [5, 10],
-              }}
-            >
-              {organization?.species?.species?.map((s) => (
-                <React.Fragment key={s.name}>
-                  <TreeSpeciesCard
-                    name={s.name}
-                    subTitle={s.desc || '---'}
-                    count={s.total}
-                  />
-                  <Box sx={{ mt: [2, 4] }} />
-                </React.Fragment>
-              ))}
-            </Box>
-            {(!organization?.species?.species ||
-              organization?.species?.species.length === 0) && (
+            {organization?.species?.species?.length ? (
```

Comment:
Can we explicit check if the length is greater than 0

```suggestion
            {organization?.species?.species?.length > 0 ? (
```